arduino/main/lib/src/ServoController.cpp

```diff
@@ -10,104 +10,97 @@
 ServoController::ServoController(
     int pwm_pin, int enable_pin, int in1_pin, int in2_pin, char duty,
     int analog_pin
-) {
-    this->pwm_pin = pwm_pin;
-    this->enable_pin = enable_pin;
-    this->in1_pin = in1_pin;
-    this->in2_pin = in2_pin;
-    this->duty = duty; // TODO: Change this ??
-    this->analog_pin = analog_pin;
-    this->direction = 0;
-}
+) : _pwm_pin(pwm_pin), _enable_pin(enable_pin), _in1_pin(in1_pin),
+    _in2_pin(in2_pin), _duty(duty), _analog_pin(analog_pin), _direction(0) { }
```

Comment:
I don't know what correct formatting is... I'm currently installing clang-format